source/_posts/2015-06-15-my-e-mail-policy.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "My email Policy"
+date: 2015-06-15 10:11:44 +1000
+comments: true
+published: true
+categories: email, pomodoro, policy
+---
+
+When I was working for [Atos](http://www.atos.net), the company's CEO started a [zero email initiative](http://atos.net/en-us/home/we-are/zero-email.html). His aim was for the employees not to use email for internal communications. I remembered what I thought about it: "It's impossible".
+Now I believe that he was right.
+This post explains how I deal with my professional email inbox.
+
+## Keep email closed
+I don't keep my email inbox open during the working hours. I open it just when I have to. I also disabled the email notification in my cellphone. That prevents me from check email whatever I see that a new email arrived.
+
+##Schedule time to check email
+I check my email twice a day, at 10:00 am and 04:00 pm. I have reminders in my calendar . That helps me organize my day and prevents me from being fooled by my weak memory.
+
+##Time box email time
+As an adopter of the [Pomodoro Technique](http://pomodorotechnique.com/) I keep track of the time I spend with email. What I noticed is that I need almost two pomodoros  to read/respond/forward/archive my emails. That means that I need almost two hours during my day only to deal with my email inbox.  That was so true when I was working for [ThoughtWorks](http://www.thoughtworks.com/). Now that I'm working for [DiUS](http://dius.com.au/) I'm trying not to sign in to so much email's lists. I aim to keep my email time in one pomodoro.
+
```

Comment:
In the second, what I wanna say actually is: "My aim is to keep my email time to one pomodoro"
Does that sound better?